drivers/entropy/entropy_cc310.c

```diff
@@ -60,11 +62,16 @@ static int entropy_cc3xx_rng_get_entropy(
 	}
 
 #else
+	size_t olen;
 	/** This is a call from a secure app, in which case entropy is
-	 *  gathered using CC3xx HW using the
+	 *  gathered using CC3xx HW using the ctr drbg features of the
```

Comment:
please use the capitalized term CTR_DRBG as this corresponds to the naming in NIST standard